Source/bmalloc/libpas/common.sh

```diff
@@ -25,7 +25,7 @@ set -e
 set -x
 
 config=Release
-sdk=macosx.internal
+sdk=macosx
```

Comment:
Can we keep this thing? We would like to build libpas with internal sdk.

Reply:
We can probably check whether we are using the internal sdk or not and adjust this based on the result?

We should ideally be able to build with both with no config changes for the user.

Reply:
We talked about this briefly in slack, but I didn't realize libpass has build scripts that run outside of an xcodebuild or Make context.

IMO, we should really replace these bespoke build scripts with something akin to the rest of WebKit, i.e. Makefiles which call into webkitdirs.pm where we have a bunch of centralized logic to compute things like which SDK you can use, etc.

In the meantime, you could use `perl -I../../../Tools/Scripts -Mwebkitdirs -e 'print XcodeOptionString()'`, which is what Make does to figure out the xcodebuild command line including SDKROOT.

Reply:
I think the context is libpas is separately independently managed, so we need to keep it work without the rest of the WebKit tree. So we should not assume that there is `../../../Tools/Scripts` script.